---
 crypto/rand/md_rand.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/crypto/rand/md_rand.c
+++ b/crypto/rand/md_rand.c
@@ -490,6 +490,7 @@ int ssleay_rand_bytes(unsigned char *buf
             goto err;
 
 #ifndef PURIFY                  /* purify complains */
+#if 0
         /*
          * The following line uses the supplied buffer as a small source of
          * entropy: since this buffer is often uninitialised it may cause
@@ -500,6 +501,7 @@ int ssleay_rand_bytes(unsigned char *buf
         if (!MD_Update(&m, buf, j))
             goto err;
 #endif
+#endif
 
         k = (st_idx + MD_DIGEST_LENGTH / 2) - st_num;
         if (k > 0) {
